draft-normington-jsonpath-latest.xml

```diff
@@ -202,24 +216,34 @@ root-selector = %x24               ; $ selects document root node
                   <name>Syntax</name>
                   <t>A dot child matcher has a key known as a dot child name or a single asterisk (<tt>*</tt>).
                   </t>
-                  <t>A dot child name corresponds to a key in a JSON object, without the surrounding double quotes (<tt>"</tt>).
+                  <t>A dot child name corresponds to a name in a JSON object.
                   </t>
                   <sourcecode type="abnf">
 matcher = dot-child             ; see below for alternatives
 dot-child = %x2E dot-child-name / ; .&lt;dot-child-name&gt;
             %x2E %x2A             ; .*
-dot-child-name = TBD
+dot-child-name = 1*(
+                   %x2D / ; -
+                   %x30-39 / ; 0-9 (core ABNF: DIGIT)
+                   %x41-5A / %x61-7A / ; A-Z / a-z (core ABNF: ALPHA)
```

Comment:
I think it would be neater to define separate rules for `DIGIT` and `ALPHA`. Then we could probably do without the "core ABNF" comments, which are slightly distracting for the general reader.

Reply:
Done!

Reply:
nit: Also, it would be nice to line up the comments vertically. :-)